__tests__/main.test.ts

```diff
@@ -55,7 +55,8 @@ function generateMilestone(
 
 const DefaultProcessorOptions: MilestoneProcessorOptions = {
   repoToken: 'none',
-  debugOnly: true
+  debugOnly: true,
+  minIssues: 3
```

Comment:
```suggestion
  minIssues: 1
```
Let's default to 1 instead

Reply:
Sure, I'll update the default value in the action manifest too 👍🏻